perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AttributesManagerBlImpl.java

```diff
@@ -1099,6 +1219,25 @@ public Attribute getAttributeById(PerunSession sess, Resource resource, Member m
 		}
 	}
 
+	@Override
+	public Attribute getAttributeById(PerunSession sess, Member member, Group group, int id) throws InternalErrorException, AttributeNotExistsException, WrongAttributeAssignmentException {
```

Comment:
This methods seems to be pretty clever (from member and group is able to return user attribute) but there is really poor javadoc about this functionality. Please, add some information to javadoc that this methdo can return member-group attribute, but also can return member or user attribute.